scripts/includes/common.sh

```diff
@@ -173,10 +173,14 @@ generate_symmetric_crypto_key_for_datastore() {
   DATASTORE_ENCRYPTION_KEY_PATH="${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}/datastore_key.json"
 
   sudo mkdir -p ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
-  sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
+
+  # If the file ${DATASTORE_ENCRYPTION_KEY_PATH} exists and is not empty, then do not generate
+  # a new key. st2-generate-symmetric-crypto-key fails if the key file already exists.
+  if [ ! -s ${DATASTORE_ENCRYPTION_KEY_PATH} ]; then
```

Comment:
Ouch,
after taking a second look at it, @warrenvw this looks like the same issue like in previous PR.

`if [ ! -s ${DATASTORE_ENCRYPTION_KEY_PATH} ]; then` might not work for non-root user which doesn't use `sudo` for check.